Phonebook.Frontend/src/app/shared/components/navigation/navigation.component.html

```diff
@@ -104,7 +103,7 @@
         <mat-toolbar color="primary">
           <mat-toolbar-row class="mat-elevation-z6">
             <div class="left app-title">
-              <button mat-button [routerLink]="['/']" *ngIf="unicornActive != true">
+              <button mat-button [routerLink]="['/']" *ngIf="firstApril != true">
```

Comment:
```suggestion
              <button mat-button [routerLink]="['/']" *ngIf="unicornActive != true">
```

Reply:
Did not change that. These changes were made in another PR and I just merged the current master branch into min before posting my PR

Reply:
Yep, nonetheless, they should not get back into master :)

Reply:
Do you had merge conflicts?

Reply:
Yes, I've had like 8 😄